### PR TITLE
Adds user page for vip users

### PR DESCRIPTION
**Pull request by timongll**
image

### CODE REVIEW

File: vip/src/pages/DepositPage/DepositPage.tsx [hunk 1]
```diff
@@ -192,16 +193,21 @@ const DepositPage = () => {
...
+  console.log("reached");
 
-  if (!vaultOption) {
+  if (!vaultOption || !auth) {
     return <Redirect to="/" />;
   }
-
-  if (!storage || !storage.includes(vaultOption)) {
-    return <Redirect to="/" />;
+  console.log("reached1");
```
Comment (leonkho) [inline vip/src/pages/DepositPage/DepositPage.tsx:203]: Console.log

File: vip/src/pages/DepositPage/DepositPage.tsx [hunk 2]
```diff
@@ -192,16 +193,21 @@ const DepositPage = () => {
...
+  if (allUserVaults().length > 0) {
+    if (!allUserVaults().includes(vaultOption)) {
+      return <Redirect to="/" />;
+    }
   }
 
+  console.log("reached2");
```
Comment (leonkho) [inline vip/src/pages/DepositPage/DepositPage.tsx:210]: Console.log

File: webapp/src/components/VIP/VIPBreakdown.tsx [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+      return colors.red;
+    default:
+      return colors.green;
+  }
+};
+
+const getAirdropTitle = (variant: ProcessedAirdropBreakdownType) => {
+  switch (variant) {
+    case "option":
+      return "Options Trader";
```
Comment (leonkho) [inline webapp/src/components/VIP/VIPBreakdown.tsx:32]: Make sure that the title is uppercased in CSS if this is capitalized here

File: webapp/src/components/Wallet/AccountStatus.tsx [hunk 4]
```diff
@@ -268,7 +274,7 @@ const MenuCloseItem = styled(MenuItem)`
 `;
 
 const AirdropMenuItem = styled(MenuItem)`
-  padding: 8px 8px;
+  padding: 4px 8px 4px 8px;
```
Comment (leonkho) [inline webapp/src/components/Wallet/AccountStatus.tsx:277]: Can be simplified to just `4px 8px`

File: webapp/src/components/VIP/VIPInfo.tsx [hunk 5]
```diff
@@ -202,7 +202,7 @@ const VIPInfo: React.FC = () => {
           <ActionButton
             className="btn"
             color={colors.primaryText}
-            onClick={() => window.open(URLS.vip)}
+            onClick={() => window.open(URLS.treasury)}
```
Comment (leonkho) [inline webapp/src/components/VIP/VIPInfo.tsx:205]: Why is this reverted?

**Review by leonkho — CHANGES_REQUESTED** [review]

File: webapp/src/components/VIP/VIPInfo.tsx (hunk 5 above)
Comment (timongll) [reply webapp/src/components/VIP/VIPInfo.tsx:205]: VIP site is not ready yet so we are sending vips to the treasury page first.

**Review by leonkho — APPROVED** [review]

**Pull request merged**